templates/dynamicdb/ldap.conf.erb

```diff
@@ -0,0 +1,52 @@
+
+# This file managed by puppet - changes will be lost
+dynamic-db "<%= @dbname %>" {
+	library "ldap.so";
```

Comment:
`ldap.so` seems like the kind of thing that will vary by platform. This is fine for now, though.